### ISSUE TITLE
merge with coyotebush's vim-pweave?

### ISSUE THREAD

**Issue by naught101** [g2boojum/vim-pweave#1]
I haven't looked indepth at this implementation, but it seems like there'd be some good opportunity for cross-pollination with https://github.com/coyotebush/vim-pweave.

@coyotebush?

**Comment by coyotebush** [comment]
At a glance, they look pretty similar to me (both are based on the `rnoweb` syntax files). This one supports ReStructuredText, mine removes some of the inapplicable Sweave-specific highlighting.

I'd be happy to see the best of both combined in one place. (I haven't even used mine since the [github-network-analysis](https://github.com/coyotebush/github-network-analysis) project.)

**Comment by g2boojum** [comment]
Yep, they're very similar.  There are some differences in naming (and file extensions), and I think Pweave has actually evolved a bit since both of these were written.

I'm fine with combining the best of both, but I probably won't get to it anytime soon.
